README.md

```diff
@@ -8,21 +8,21 @@
 > [issues](https://github.com/GoogleCloudPlatform/functions-framework-dart/issues)
 > for bugs.
 
-|Functions Framework|Unit Tests|Lint Test|Conformance Tests|
-|---|---|---|---|
-[Dart][ff_dart]| [![][ff_dart_unit_img]][ff_dart_unit_link] | [![][ff_dart_lint_img]][ff_dart_lint_link] | [![][ff_dart_conformance_img]][ff_dart_conformance_link] |
+| Functions Framework | Unit Tests                                 | Lint Test                                  | Conformance Tests                                        |
+| ------------------- | ------------------------------------------ | ------------------------------------------ | -------------------------------------------------------- |
```

Comment:
I'd revert this. 3 en dashes is common and more maintainable.

Reply:
done